kuber/kube.go

```diff
@@ -18,12 +18,12 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 
-	kbeta2 "k8s.io/api/apps/v1beta2"
+	appsV1 "k8s.io/api/apps/v1"
 	kbeta1 "k8s.io/api/batch/v1beta1"
 	kv1 "k8s.io/api/core/v1"
 	kmeta "k8s.io/apimachinery/pkg/apis/meta/v1"
-	beta2client "k8s.io/client-go/kubernetes/typed/apps/v1beta2"
-	kapps "k8s.io/client-go/kubernetes/typed/apps/v1beta2"
+	v1Client "k8s.io/client-go/kubernetes/typed/apps/v1"
+	kapps "k8s.io/client-go/kubernetes/typed/apps/v1"
```

Comment:
These 2 declarations are duplicates with different names. Remove one of them.

Reply:
ok

Reply:
i tested it on version 1.15 and 1.16